### PR TITLE
Improve ALTER INDEX and UNSPLIT AT. Doc UNSPLIT ALL.

### PR DESCRIPTION
**Pull request by stbof**
DOC-1268
DOC-2492

### CODE REVIEW

**Review by ajwerner — APPROVED** [review]
:lgtm:

> Reviewed 3 of 4 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14660#-:-N925IYE0zp1fs60upVF:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @stbof)

File: v22.1/unsplit-at.md [hunk 1]
```diff
@@ -214,6 +100,8 @@ Now unsplit the index to remove the split enforcements:
 (3 rows)
 ~~~
 
+You can see the split's expiration date in the `split_enforced_until` column. The [`crdb_internal.ranges`](crdb-internal.html) table also contains information about ranges in your CockroachDB cluster, including the `split_enforced_until` column.
```
Comment (taroface) [inline v22.1/unsplit-at.md:103]: `crdb_internal` is marked as a `Reserved` interface in the [support policy](https://www.cockroachlabs.com/docs/v22.1/api-support-policy.html), and the `ranges` table is not listed as one of the "ok for production" exceptions in the `crdb_internal` doc. IMO we should not bring it up, but happy to defer to any eng who disagrees.

```suggestion
You can see the split's expiration date in the `split_enforced_until` column.
```

File: v22.1/unsplit-at.md [hunk 2]
```diff
@@ -214,6 +100,8 @@ Now unsplit the index to remove the split enforcements:
...
+
 {% include_cached copy-clipboard.html %}
 ~~~ sql
 > SELECT range_id, start_pretty, end_pretty, split_enforced_until FROM crdb_internal.ranges WHERE table_name='rides';
```
Comment (taroface) [inline v22.1/unsplit-at.md:107]: Given the above, we should remove this snippet too.

File: v22.1/split-at.md [hunk 3]
```diff
@@ -265,7 +265,7 @@ You can specify the time at which a split enforcement expires by adding a `WITH
 (3 rows)
 ~~~
 
-You can see the split's expiration date in the `split_enforced_until` column. The `crdb_internal.ranges` table also contains information about ranges in your CockroachDB cluster, including the `split_enforced_until` column.
+You can see the split's expiration date in the `split_enforced_until` column. The [`crdb_internal.ranges`](crdb-internal.html) table also contains information about ranges in your CockroachDB cluster, including the `split_enforced_until` column.
```
Comment (taroface) [inline v22.1/split-at.md:268]: Same comment as below (sorry, saw that one first).

File: v22.1/split-at.md [hunk 4]
```diff
@@ -151,7 +151,7 @@ Suppose that you want MovR to offer ride-sharing services, in addition to vehicl
 );
 ~~~
 
-The table's compound primary key is on the `city` and `dl` columns. Note that the table automatically generates an `id` and a `dl` [using supported SQL functions](functions-and-operators.html), if they are not provided.
+The table's compound primary key is on the `city` and `dl` columns. Note that the table automatically generates an `id` and a `dl` value [using supported SQL functions](functions-and-operators.html), if they are not provided.
```
Comment (taroface) [inline v22.1/split-at.md:154]:
```suggestion
The table's compound primary key is on the `city` and `dl` columns. Note that the table automatically generates an `id` and a `dl` value [using supported SQL functions](functions-and-operators.html) if they are not provided.
```

**Review by taroface — APPROVED** [review]
LGTM, with a nit and a question on `crdb_internal`.

File: v22.1/unsplit-at.md (hunk 1 above)
Comment (stbof) [reply v22.1/unsplit-at.md:103]: I asked @kevin-v-ngo about that and he said it was OK to leave it in.

**Pull request merged**